README.md

```diff
@@ -9,7 +9,42 @@ export OPERAND_IMAGE="quay.io/app-sre/cincinnati:2873c6b"
 operator-sdk run --local
 ```
 
+## Run latest
+
+```
+oc apply -k deploy
+```
+
+## Deploy using Operator Lifecycle Manager (OLM)
+
+You will first need to get an access token from quay.io and use it to create a
+Secret:
+
+```
+TOKEN=$(curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '
+{
+    "user": {
+        "username": "'"${QUAY_USERNAME}"'",
+        "password": "'"${QUAY_PASSWORD}"'"
+    }
+}' | jq -r '.token')
+
+oc create secret generic quay-registry-cincinnati -n openshift-marketplace --from-literal="token=$TOKEN"
+```
+
+Then make cincinnati-operator visible to OLM/OperatorHub:
+
+```
+oc apply -k deploy/olm/marketplace
```

Comment:
Are there additional steps needed for OperatorHub deployments to work? I tried applying the marketplace manifests and see this error message in the cincinnati CatalogSource:  
''''
message: The OperatorSource endpoint returned an empty manifest list
''''